README.md

```diff
@@ -1 +1,98 @@
-# books-website
\ No newline at end of file
+# books-website
+
+# Project Name : Awsome Books ♻️
+
+This is  Pair to Pair project to help you find and remember the best books to read.
+
+# Screenshots
+- Desktop Layout
+
+![Desktop_Layout](assets/img/desktop.png)
+
+- Mobile Layout
+
+![Mobile_Layout](assets/img/mobile.png)
+
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML 
+- CSS
+- Javascript
+- VScode
+- Git & GitHub
+
+## Live Demo link
+
+[Live Demo Link](https://PinkMoon25.github.io/books-website/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+have a working git on your localmachine.
+## Install
+To get a working local copy up and running follow these simple steps
+- open terminal
+- clone this project with the command:
+
+
+```
+$ git clone https://github.com/PinkMoon25/books-website.git
+```
+- Finally, run the next command in the terminal to see the web app in your local browser, or also, you can use Live Server in Visual Studio Code.
+```
+$ npm start
+```
+### Usage
+For anyone that whats to learn html & css & Js
+### Run tests
+N/A
+### Deployment
+N/A
+
+
+## Authors
+
+👤 **Author 1 **
+
+- GitHub: [@PinkMoon25](https://github.com/PinkMoon25/)
+- Twitter: [@twitterhandle](https://twitter.com/---)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/---/)
+
+👤 **Author 2 **
```

Comment:
- [ ] [OPTIONAL] Kindly change the name of the authors with your name instead of `Author 1` and `Author 2`

Reply:
hi, @gemmen29 we changed the author's name to ours.